src/Rules/Rule.cs

```diff
@@ -19,17 +20,28 @@ abstract class Rule : IRule
     {
         public RuleInfo Info { get; private set; }
         public Condition Condition { get; }
+        protected static Condition DefaultExcludedCondition => IsChromiumContent;
```

Comment:
Exposing this to derived classes gives us a single place to modify the default, then the production constructors of the derived classes just pick up the new value without requiring any additional code change